homeassistant/components/shelly/entity.py

```diff
@@ -350,10 +336,6 @@ async def async_added_to_hass(self) -> None:
         """When entity is added to HASS."""
         self.async_on_remove(self.coordinator.async_add_listener(self._update_callback))
 
-    async def async_update(self) -> None:
-        """Update entity with latest info."""
-        await self.coordinator.async_request_refresh()
-
```

Comment:
This is already implemented in the base `CoordinatorEntity` class and is redundant:
https://github.com/home-assistant/core/blob/3f0be3dfba96d52d24d7e83c279be4c88322a767/homeassistant/helpers/update_coordinator.py#L363-L372